tests/pd/test_rpc_client.rs

```diff
@@ -24,12 +28,15 @@ use tikv::pd::{PdClient, RpcClient, validate_endpoints, Error as PdError, Region
 use super::mock::mocker::*;
 use super::mock::Server as MockServer;
 
+static PORT: AtomicUsize = AtomicUsize::new(52100);
```

Comment:
can we use the way that listens port 0, gets a random port and then uses it?

I worry fixed PORT may be conflicted with other tests in raftstore which uses random port.